src/ydata_synthetic/synthesizers/regular/cramergan/model.py

```diff
@@ -171,7 +171,7 @@ def save(self, path):
         super().save(path)
 
 
```

Comment:
Noticed that the model was already imported in this script.
Just cleans up code, no changes come from it.

Reply:
Please do not include changes in other models in the same PR as TSCGAN. 

Have this moved into a different branch.